src/index.js

```diff
@@ -184,11 +184,10 @@ const exported = {
      * @var {string} workerUrl
      * @returns {string} A URL hosting a JavaScript bundle for mapbox-gl's WebWorker.
      * @example
-     * <script src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl-csp.js'></script>
-     * <script>
+     * // <script src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl-csp.js'></script>
+     * // <script>
```

Comment:
Why do these require `//`? Seems like an odd choice to me

Reply:
It's our only html example and the html fails linting. I agree the `//` isn't good. 4260e89 fixes this by disabling linting for examples with `<script>` in them.